### PR TITLE
Add basic CI with Github Actions

### PR DESCRIPTION
**Pull request by xhochy**
This still skips a lot of the stores but already gives us a basis to work again on this project.

### CODE REVIEW

**Comment by xhochy** [comment]
cc @siboehm @mlondschien

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+on: [push]
+
+jobs:
+  unittest:
+    name: Unit tests - ${{ matrix.PYTHON_VERSION }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        PYTHON_VERSION: ["3.7", "3.8"]
```
Comment (fjetter) [inline .github/workflows/ci.yml:11]: out of curiosity, did you try 3.9 already?

**Review by fjetter — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (xhochy) [reply .github/workflows/ci.yml:11]: No, I would like to do that in a separate PR. My main intention with this PR is to get this going so one can work in parallel on the various issues.

File: README.rst [hunk 2]
```diff
@@ -1,33 +1,28 @@
...
 ------------
-simplekv is `available on PyPI <http://pypi.python.org/pypi/simplekv/>`_ and
+minimalkv is `available on PyPI <http://pypi.python.org/pypi/minimalkv/>`_ and
 can be installed through `pip <http://pypi.python.org/pypi/pip>`_::
 
-   pip install simplekv
+   pip install minimalkv
    
-or via ``conda`` on `conda-forge <https://github.com/conda-forge/simplekv-feedstock>`_::
+or via ``conda`` on `conda-forge <https://github.com/conda-forge/minimalkv-feedstock>`_::
```
[the remark this answers is not in the document]
Comment (xhochy) [reply README.rst:16]: Done

File: docs/development.rst [hunk 3]
```diff
@@ -4,6 +4,6 @@ Development
 Github
 ======
 All official development of the library takes place on `GitHub
-<https://github.com/mbr/simplekv>`_. Comments, bug reports and patches are
+<https://github.com/data-engineering-collective/minimalkv>`_. Comments, bug reports and patches are
```
[the remark this answers is not in the document]
Comment (xhochy) [reply docs/development.rst:7]: Done.

File: README.rst [hunk 4]
```diff
@@ -1,33 +1,28 @@
...
-*simplekv* is an API for very basic key-value stores used for small, frequently
+*minimalkv* is an API for very basic key-value stores used for small, frequently
 accessed data or large binary blobs. Its basic interface is easy to implement
 and it supports a number of backends, including the filesystem, SQLAlchemy,
 MongoDB, Redis and Amazon S3/Google Storage.
 
 Installation
 ------------
-simplekv is `available on PyPI <http://pypi.python.org/pypi/simplekv/>`_ and
+minimalkv is `available on PyPI <http://pypi.python.org/pypi/minimalkv/>`_ and
```
[the remark this answers is not in the document]
Comment (xhochy) [reply README.rst:11]: Just a placeholder release.

**Review by mlondschien — APPROVED** [review]

**Pull request merged**